src/communicator.ts

```diff
@@ -18,6 +19,8 @@ export interface CommunicatorContext extends DBOSContext {
   // These fields reflect the communictor's configuration.
   readonly retriesAllowed: boolean;
   readonly maxAttempts: number;
+  getConfiguredClass<C extends InitConfigMethod>(_cls: C): ConfiguredClass<C, Parameters<C['initConfiguration']>[1]>;
```

Comment:
Let's decide if we like getConfiguredClass at all before we decide if we need it here.

Reply:
I think `getConfiguredClass` is necessary for workflow context because it needs get the configured class to `invoke` other functions. But what's the use case for communicator and transaction contexts?